atlasdb-impl-shared/src/test/resources/lockwatch-event-cache-output/cacheClearedOnSnapshotUpdate/event-cache-2.json

```diff
@@ -36,7 +36,11 @@
       }
     },
     "livingVersions" : {
-      "7" : [ 123, 1255 ]
+      "Sequence{value=7}" : [ {
+        "start-ts" : 123
+      }, {
+        "start-ts" : 1255
+      } ]
```

Comment:
Not the most compact, and I don't know why sequence and start timestamp serialise differently, but whatever. It's now clear what's going on internally.

Reply:
They do serialise the same, but both `TreeMap` and `TreeMultimap` do some weird stuff. Since this is only for tests anyways, it's fine